tools/tests/test_lint.py

```
@@ -3717,3 +3735,748 @@ def test_git_ignored_filters_an_ordinary_tree(tmp_path):
+    for rel in ("docs/reviews.jsonl", "docs/recorded-findings.jsonl",
+                "docs/ledger.jsonl", "docs/architecture/adr/ADR-001.md"):
+        path = tmp_path / rel
+        path.parent.mkdir(parents=True, exist_ok=True)
+        path.write_text(defect, encoding="utf-8", newline="")
+    (tmp_path / "docs" / "live.md").write_text(defect, encoding="utf-8", newline="")
+    findings = lint.check_hollow_code_span(tmp_path)
+    assert len(findings) == 1, "only the file that is not a record is reported"
+    assert "docs/live.md" in findings[0]
+
+
+def test_the_skip_lists_hold_the_paths_the_doctrine_names(tmp_path):
```

Ruff (0.16.2)

[warning] Unused function argument: `tmp_path`

(ARG001)